### ISSUE TITLE
[ Feature ] Search for Emojis

### ISSUE THREAD

**Issue by rickumtickum** [dslul/openboard#197]
It would be awesome if there is a search option for emojis. 
Like typing in cry would show us crying emoji. 
Search only based on the keyword entered in search box when in emoji screen. 
Thanks!

**Comment by wrapper** [comment]
Yes please! The emjoi panel is the thing that makes this keyboard particulary difficult to use

**Comment by hermann-san** [comment]
good idea, or even emoji prediction. I.e. you are writing a chat message and while you type "cry" within the text, a cry emoji automatically come's in the word corrections like this.
![Auswahl_027](https://user-images.githubusercontent.com/7679654/99115111-35365a80-25f2-11eb-9c68-a372a874d1ca.png)

**Comment by dspacejs** [comment]
+1 not having this feature is really annoying.

**Comment by MonsieurBibo** [comment]
+1  I would really like this to happen

**Comment by derei** [comment]
I concur, being able to search for emojis is such a great feature that's missing from openboard. It should be seriously considered.

**Comment by L0stLink** [comment]
emoji prediction would be great but even being able to select an emoji using slack-like `:` prefixed syntax would be highly appreciated, Thank you for all the work on this BTW.

**Comment by Matmut7** [comment]
I'd be glad to work on this (a core search feature first, then search bar / predictions / slack-line selection) but I never contributed to OpenBoard, do you think this feature is affordable for me ? I know Java but not that much of Android dev.

**Comment by woolmonkey** [comment]
Yes at least an emoji search then hopefully that can change to a emoji suggestion.

**Comment by Zipdox** [comment]
I really really want emoji search indeed

**Comment by rramnel** [comment]
This would be a great feature to have

**Comment by Sitavi** [comment]
I'm agree with that, would be really great to have this feature. That's why anysoftkeyboard has an advantage for me

**Comment by l0stlink** [comment]
Hi all, please limit conversation to only sharing unique suggestions or related topics, please use :+1: / :-1: to express your support. A lot of people are subscribed to this thread and whenever someone enters a comment they all receive emails form it. Thanks.

**Comment by jnnkB** [comment]
I renamed this to Emoji Prediction since emoji search would be a duplicate of #35
